code/modules/antagonists/vampire/vamp_datum.dm

```diff
@@ -154,6 +154,32 @@
 
 #undef BLOOD_GAINED_MODIFIER
 
+/**
+ * Remove the vampire's current subclass and add the specified one.
+ *
+ * Arguments:
+ * * new_subclass_type - a [/datum/vampire_subclass] typepath
+ */
+/datum/antagonist/vampire/proc/change_subclass(new_subclass_type)
+	if(isnull(new_subclass_type))
+		return
+	clear_subclass(FALSE)
+	add_subclass(new_subclass_type)
```

Comment:
I am deliberating if this should create a DB log. currently we log whenever a vampire adds a subclass. might be a good idea to phase out admins skewing the data.

Reply:
Oh yeah, I can add an arg to `add_subclass` for creating logs and set it to false here.